Sitenotice.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-if ( $wmgSiteNoticeOptOut ) {
+/* if ( $wmgSiteNoticeOptOut ) {
```

Comment:
Should not comment out this if statement.

Reply:
@Universal-Omega Oh okay, where do you want me to move it to then?

Reply:
To only the second portion where the sitenotice actually is.

Reply:
> To only the second portion where the sitenotice actually is.

Okay, doing that now, and hoping I get it right.

Reply:
👍 **Done**